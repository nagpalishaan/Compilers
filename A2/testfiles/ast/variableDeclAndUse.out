Program
  List
    Function
      IdDecl ID="a"
      Opt
      List
        DeclareVar
          IdDecl ID="a"
          Opt
        DeclareVar
          IdDecl ID="b"
          Opt
        AssignVar
          IdUse ID="a"
          Numeral NUMERAL="2"
        AssignVar
          IdUse ID="b"
          Numeral NUMERAL="3"
