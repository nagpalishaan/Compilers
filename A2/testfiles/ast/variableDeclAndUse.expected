Program
  List
    Function
      IdDecl ID="a"
      Opt
      List
        DeclareVar
          IdDecl ID="a"
          Opt
        DeclareVar
          IdDecl ID="b"
          Opt
        AssignVar
          IdDecl ID="a"
          Numeral NUMERAL="2"
        AssignVar
          IdDecl ID="b"
          Numeral NUMERAL="3"
